zipline/data/data_portal.py

```diff
@@ -15,6 +15,7 @@
 from operator import mul
 
 import bcolz
+from cachetools import lru_cache
```

Comment:
this is available in functools32, if we add this we should also update `memoize_last` in zipline.utils.memoize to be implemented as `lru_cache(1)`